hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackHelper.java

```diff
@@ -145,6 +163,12 @@ public ListingBasedRollbackHelper(HoodieTableMetaClient metaClient, HoodieWriteC
       }
       return false;
     };
-    return fs.listStatus(FSUtils.getPartitionPath(config.getBasePath(), partitionPath), filter);
+    return metaClient.getFs().listStatus(FSUtils.getPartitionPath(config.getBasePath(), partitionPath), filter);
```

Comment:
Let's extract `listStatus` into the parent method, and make both `listFilesToBeDeleted` and `getFilesFromCommitMetadata` static